source/components/ansible-setup.rst

```diff
@@ -59,6 +59,19 @@ Allan have access to it. Stash the password in file named
 You will also need sudo access on the hosts you want to change if that change
 requires privilege escalation.
 
+.. note::
```

Comment:
I'm not sure that a blank line is _required_ here but I suggest leaving one for consistency with directives that do require them.

Reply:
I have added it, like @danielballan I prefer to be consistent even if it isn't necesary